examples/2_intermediate/mta.ipynb

```diff
@@ -0,0 +1,495 @@
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "\u001b[0;31mType:\u001b[0m        property\n",
+       "\u001b[0;31mString form:\u001b[0m <property object at 0x7f37ac4d25c0>\n",
+       "\u001b[0;31mDocstring:\u001b[0m   Get the list of subway trips from the GTFS-realtime feed. Returns a list of `Trip` objects"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "feed.trips?"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "3bf0fe58-ced8-49a7-84bb-09b9abdb41ee",
+   "metadata": {},
+   "source": [
+    "In our case, we will (for simplicity) filter the trips to collect information only about the 1, 2 and 3 trains, and we will start with trains going through 34 St-Penn Station."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "id": "429163be-6a02-4a14-b1f5-5f5016213af0",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "[{\"069950_1..N03R\", STOPPED_AT 133N @12:52:08},\n",
+       " {\"070550_1..N03R\", STOPPED_AT 137N @12:51:41},\n",
+       " {\"071150_1..N03R\", IN_TRANSIT_TO 139N @12:52:23},\n",
+       " {\"067150_2..N09R\", STOPPED_AT 128N @12:51:58},\n",
+       " {\"067950_2..N01R\", IN_TRANSIT_TO 132N @12:52:26},\n",
+       " {\"068750_2..N09R\", IN_TRANSIT_TO 230N @12:52:27},\n",
+       " {\"069550_2..N01R\", STOPPED_AT 235N @12:52:05},\n",
+       " {\"070350_2..N09R\", STOPPED_AT 241N @12:51:56},\n",
+       " {\"071150_2..N01R\", IN_TRANSIT_TO 246N @12:52:22},\n",
+       " {\"066500_3..N01R\", STOPPED_AT 132N @12:51:45},\n",
+       " {\"067100_3..N01R\", STOPPED_AT 137N @12:51:58},\n",
+       " {\"067900_3..N01R\", INCOMING_AT 230N @12:52:26},\n",
+       " {\"068700_3..N01R\", IN_TRANSIT_TO 233N @12:52:22},\n",
+       " {\"069500_3..N01R\", INCOMING_AT 239N @12:52:25},\n",
+       " {\"070300_3..N01R\", STOPPED_AT 252N @12:51:38},\n",
+       " {\"071150_3..N01R\", INCOMING_AT 256N @12:52:06}]"
+      ]
+     },
+     "execution_count": 6,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# This cell can be run multiple times, and data will be refreshed every 30s\n",
+    "feed.refresh()\n",
+    "trains = feed.filter_trips(underway=True, headed_for_stop_id=['128', '128N'])\n",
+    "trains"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "e52dd161-b878-4d6f-b4a0-bda3a6f35967",
+   "metadata": {},
+   "source": [
+    "We can also show this data in a human-readable way."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "id": "071a82d9-9da6-484f-951c-2c0bbc26f0f6",
+   "metadata": {
+    "scrolled": true
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Northbound 1 to Van Cortlandt Park-242 St, departed origin 11:39:30, Currently STOPPED_AT Christopher St-Sheridan Sq, last update at 12:52:08\n",
```

Comment:
Note that these are Northbound only since we only give `128N` as a stop, if we provide `128S` we get both directions.